release.nix

```diff
@@ -0,0 +1,5 @@
+{ devices ? [ "google-blueline" ] }:
+
+with import <nixpkgs> {};
+
+lib.genAttrs devices ( device: (import ./. { inherit device; }).build.android-bootimg)
```

Comment:
Achtung! Not all devices are android-based!

Though re-reading my system types, I see we need a couple well-known universal attributes. (This is on my to implement)

 * `disk-image` (already present in depthcharge) a full disk image with `stage-2` and `stage-1`. Not applicable to all devices
 * `stage-2`, basically `system.img` for android-based devices. Most often not possible to cross-compile
 * `stage-1`, could be called initramfs or initrd, but those terms are confused and confusing.

This way, we could always try, by default, and build stage-1, stage-2 and disk-image on release, both as cross (on x86_64) or native (aarch64).

Any suggestions for better well-known attribute names? I'm thinking `stage-1` and `stage-2` without other qualified could clash... Maybe something like `stage-n.img`?